roles/custom/matrix-rageshake/templates/systemd/matrix-rageshake.service.j2

```diff
@@ -0,0 +1,42 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix Rageshake Service
+{% for service in matrix_rageshake_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_rageshake_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ devture_systemd_docker_base_systemd_unit_home_path }}"
+ExecStartPre=-{{ devture_systemd_docker_base_host_command_sh }} -c '{{ devture_systemd_docker_base_host_command_docker }} kill matrix-rageshake 2>/dev/null || true'
+ExecStartPre=-{{ devture_systemd_docker_base_host_command_sh }} -c '{{ devture_systemd_docker_base_host_command_docker }} rm matrix-rageshake 2>/dev/null || true'
+
+ExecStart={{ devture_systemd_docker_base_host_command_docker }} run --rm --name matrix-rageshake \
+			--log-driver=none \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			--cap-drop=ALL \
+			--read-only \
+			--tmpfs /tmp \
+			--network={{ matrix_docker_network }} \
+			-p 9110:9110 \
```

Comment:
Why expose port `9110`?

Reply:
Don't we need that? I am honestly not sure I fully understand the complicated inner setup and interactions between all these services yet (and how to test changes properly - I often found myself in the situation that my changes weren't properly applied and/or systemd or the service not restarted with the new configuration) ...

If you say that from looking at the other files, this isn't needed, I happily remove it.

Reply:
In the default configuraiton (matrix-nginx-proxy as a reverse proxy.. or Traefik as a reverse-proxy), none of the services expose ports locally at all.